docs/usage_guide.rst

```diff
@@ -594,14 +594,21 @@ At the moment there is no compulsory API key. But if you don't use your own node
 Cointracking.info
 ====================
 
-You can also import data from `cointracking.info <https://cointracking.info/>`_ into rotki by clicking on "Import Data" on the left sidebard and then following the instructions.
+You can also import data from `cointracking.info <https://cointracking.info/>`_ into rotki by clicking on "Import Data" on the left sidebar and then following the instructions.
 
 rotki can import any trade CSV data exported from cointracking.info. But in general it's not recommended to utilize cointracking as their exported data are missing a lot of information.
 
 .. image:: images/sc_data_import.png
    :alt: Importing data from cointracking.info
    :align: center
 
+Binance.com
+=============
+
+You can import data from `binance.com <https://binance.com/>`_ into rotki by clicking on "Import Data" on the left sidebar and then following the instructions.
+
+rotki can import a CSV data exported from Binance. But due to Binance's csv format some data may be not importable. You will see warnings if this will happen.
+
```

Comment:
I would add here that the import functionality can import more things than the API provides

Reply:
Aha, yep, sounds cool

Reply:
yeah please do add this distinction

Reply:
Sure, added!